docs: Ways to upload an API (#1615)
keptn/keptn.github.io: content/docs/1.0.x/reference/api/index.md

        caption="Keptn Swagger API documentation - Example"
        width="700px">}}

 ## Uploading an API file to a Keptn project 

>  * It can be done in any of the two ways :

Reviewer: ```suggestion
 * An API file can be uploaded to a project in either of the following ways :
```

>     * Upload it directly to your git repo.

Reviewer: ```suggestion
    * Upload the file directly to your git repo.
```

>     * Use the resource-service API for uploading with the example snippted from above. 

Reviewer: ```suggestion
    * Use the `resource-service` API for uploading, as shown in the example snippet from above. 
```
Any way to link to this snippet to make it less vague?
